spec/visual/samples/llvm

```diff
@@ -71,3 +71,14 @@ second_end:
                                        ; control-equivalent to %end, so this is
                                        ; well-defined (ignoring earlier undefined
                                        ; behavior in this example).
+
+; copied from https://llvm.org/docs/LangRef.html#attribute-groups
```

Comment:
We're trying to avoid including copyrighted work in the code samples unless permitted. I had a look on the documentation website and couldn't see anything about terms of use.

Could you change these a little bit to be different from the examples on that page?

Reply:
It looks like the documentation is stored in the LLVM [main repository](https://github.com/llvm-mirror/llvm/blob/3709f6b81ea8e94f6a6b26d015d9216c9435edce/docs/LangRef.rst) which is available under the terms of the [Apache 2.0 license](https://github.com/llvm-mirror/llvm/blob/master/LICENSE.TXT) apart from certain third party tools but that do not concern the docs.

Anyway, the other snippets are all pasted from this page. Do you want me to update all the examples or do you want to leave it as is ?

Reply:
Hmmm. Let me have a chat with the other maintainers and get back to you!

Reply:
The way you'd noted the source before is sufficient. Apologies for the confusion. Nothing further required on this front.